habitat/sims/pyrobot/pyrobot.py

```diff
@@ -0,0 +1,281 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Any
+
+import numpy as np
+from gym import Space, spaces
+
+import pyrobot
+from habitat.core.registry import registry
+from habitat.core.simulator import (
+    BumpSensor,
+    Config,
+    DepthSensor,
+    RGBSensor,
+    SensorSuite,
+    Simulator,
+)
+from habitat.core.utils import try_cv2_import
+
+cv2 = try_cv2_import()
+
+
+def _locobot_base_action_space():
+    return spaces.Dict(
+        {
+            "go_to_relative": spaces.Box(low=-np.inf, high=np.inf, shape=(3,)),
+            "go_to_absolute": spaces.Box(low=-np.inf, high=np.inf, shape=(3,)),
+        }
+    )
+
+
+def _locobot_camera_action_space():
+    return spaces.Dict(
+        {
+            "set_pan": spaces.Box(low=-np.inf, high=np.inf, shape=(1,)),
+            "set_tilt": spaces.Box(low=-np.inf, high=np.inf, shape=(1,)),
+            "set_pan_tilt": spaces.Box(low=-np.inf, high=np.inf, shape=(2,)),
+        }
+    )
+
+
+ACTION_SPACES = {
+    "LOCOBOT": {
+        "BASE_ACTIONS": _locobot_base_action_space(),
+        "CAMERA_ACTIONS": _locobot_camera_action_space(),
+    }
+}
+
+
+@registry.register_sensor
+class PyRobotRGBSensor(RGBSensor):
+    def __init__(self, config):
+        super().__init__(config=config)
+
+    def _get_observation_space(self, *args: Any, **kwargs: Any):
+        return spaces.Box(
+            low=0,
+            high=255,
+            shape=(self.config.HEIGHT, self.config.WIDTH, 3),
+            dtype=np.uint8,
+        )
+
+    def get_observation(self, robot_obs):
```

Comment:
We need to keep sensor's method tolerant to new arguments that can be required by new added sensors: 
```suggestion
    def get_observation(self, robot_obs, *args: Any, **kwargs: Any):
```
Otherwise it will start throwing errors.